lavamoat/browserify/beta/policy.json

```diff
@@ -3748,6 +3748,121 @@
         "setTimeout": true
       }
     },
+    "firebase": {
```

Comment:
A good chunk of firebase permissions.

It sucks that the npm module is a "large" npm module, when realistically we only use App and Messaging.

These do get tree shaken, but guessing lavamoat policies run on the node modules?